presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -198,3 +198,61 @@ Examples::
     SELECT * FROM (VALUES ('a%c'), ('%cd'), ('cde')) AS t (name)
     WHERE name LIKE '%#%%' ESCAPE '#'
     --returns 'a%c' and  '%cd'
+
+
+Row comparison: IN
+------------------
+
+The IN comparison operator in SQL is used to compare a value with a list of literal values that have been specified.
+The IN operator returns true if the value matches any of the literal values in the list. The IN operator can be
```

Comment:
```suggestion
The IN operator returns TRUE if the value matches any of the literal values in the list. The IN operator can be
```
You capitalize TRUE in the OR paragraph: "It returns a record if any of the conditions separated by OR is TRUE."
Suggest capitalize here for consistency.